### ISSUE TITLE
The keptn.sh webiste is still being blocked by my ISP

### ISSUE THREAD

**Issue by motdde** [keptn/keptn.github.io#737]
It would be nice to have the Keptn website build on github.io

**Comment by jetzlstorfer** [comment]
Hi @motdde , could you please give more details to the issue? I can not see a clear connection between the issue title and your description.
Your issue is that you can not access https://keptn.sh ?

**Comment by christian-kreuzberger-dtx** [comment]
keptn.sh is hosted via netlify (DNS is managed via Amazon).
I've seen some others complaining about having users that are unable to reach their website:
* https://community.netlify.com/t/not-everyone-can-open-my-website-why/30539
* https://community.netlify.com/t/domain-is-not-loading-it-was-fine-last-week-what-happened/30455/8
* https://community.netlify.com/t/how-to-know-if-an-ip-is-blacklisted/8600

We are unaware of any problems on our side (at least you are the first one indicating that there is a problem). Most likely there is not much we can do, other than changing our hosting provider (and that's **not** something we can do within minutes).

If you want, you can still take a look at the website by cloning it and running it locally: https://github.com/keptn/keptn.github.io#run-locally

Other then that, please get in touch with your ISP or netlify directly.

**Comment by motdde** [comment]
Thanks @jetzlstorfer 

I have tested on four ISP's, and the Keptn site is available on only one of them, which happens to be the most expensive one. I was suggesting that the website be hosted on the GitHub page using the github.io domain.

I think the issue is with the newer TLD's (.sh .io etc). There are also some sites with these TLD's that are not accessible in my region.

**Comment by jetzlstorfer** [comment]
Hi @motdde - we refer to the Keptn website everywhere as https://keptn.sh and therefore it would be problematic for us to host it now on the github.io domain (as you said, it is also one of the newer TLDs that might cause some issues due to the `.io`) 
Are you still facing the issue or was this resolved now by your ISP?

**Comment by motdde** [comment]
Hello @jetzlstorfer,

It works fine on my new ISP. Thanks for checking in.
